cli/src/action/database.rs

```diff
@@ -23,7 +23,7 @@ use splinter::biome::credentials::database::run_migrations as run_biome_credenti
 #[cfg(feature = "database-migrate-biome-notifications")]
 use splinter::biome::notifications::database::run_migrations as run_biome_notifications_migrations;
 #[cfg(feature = "database-migrate-biome-users")]
-use splinter::biome::users::database::run_migrations as run_biome_users_migrations;
+use splinter::biome::user::database::run_migrations as run_biome_user_migrations;
```

Comment:
There's some import statements referencing `users` at 
`libsplinter/src/biome/credentials/rest_resources.rs` and `libsplinter/src/biome/rest_api/mod.rs`

Reply:
Fixed two instances of this.